src/main/io/serial_cli.c

```diff
@@ -1697,11 +1697,14 @@ static void printRxRange(uint8_t dumpMask, master_t *defaultConfig)
             channelRangeConfigurationDefault->min,
             channelRangeConfigurationDefault->max
         );
+	bufWriterFlush(cliWriter);
         cliDumpPrintf(dumpMask, equalsDefault, format,
```

Comment:
Interesting that this appears to happen only for this command. There are ~ 30 other occurrences of `cliDefaultPrintf` directly followed by `cliDumpPrintf` (and the `cliDefaultPrintf` only ever produces output when `[command] showdefaults` is used). Have you checked if any of the other occurrences shows the same problem with your setup? It might be worth adding the `bufWriterFlush` to the `...Printf` functions.